src/kuzya.h

```diff
@@ -173,6 +175,8 @@ private slots:
         QFile *file;
         QString language; //curren programing language
         QsciScintilla* textEditor;
+        QTabWidget *tabWidget;
+        QTreeWidget* mTreeViewWidget;
```

Comment:
I think, it would be more convenient to name mTreeViewWidget like as mWatchLocalsWidget or something else. Everyone will know basic things about this element by his name. Everyone know that it's a treeWidget by type of pointer.